packages/mdc-fab/_mixins.scss

```diff
@@ -50,6 +50,10 @@
   @include mdc-theme-prop(color, $color);
 }
 
+@mixin mdc-fab-extended-fluid {
```

Comment:
What's the thought process behind this being a mixin rather than just another modifier class? Seems like this could be done without requiring Sass.

Reply:
We want user to be able to use this mixin within CSS media query.